mlir/include/mlir/Dialect/MPI/IR/MPIOps.td

```diff
@@ -0,0 +1,129 @@
+//===- MPI.td - Message Passing Interface Ops ---------*- tablegen -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef MPI_OPS
+#define MPI_OPS
+
+include "mlir/Dialect/MPI/IR/MPIBase.td"
+
+class MPI_Op<string mnemonic, list<Trait> traits = []> :
+    Op<MPI_Dialect, mnemonic, traits>;
+
+//===----------------------------------------------------------------------===//
+// InitOp
+//===----------------------------------------------------------------------===//
+
+def MPI_InitOp : MPI_Op<"init", [
+    
+  ]> {
+  let summary = "Initialize the MPI library, equivalent to `MPI_Init(NULL, NULL)`";
+  let description = [{
+    This operation must preceed most MPI calls (except for very few exceptions,
+    please consult with the MPI specification on these).
```

Comment:
Nice, much better. Maybe add a link to the specification? I know you linked to it before already, but this feels like something where people might only read this very part and then having the link right here will be very nice.

Reply:
Maybe you could say somethign like "this sets up the context" or "initializes the environment", whatever fits best. Obviously, you don't need to repeat all of the docu, but a quick one or two sentence might give enough insight to most people to not have to check the MPI docu.